src/App.tsx

```diff
@@ -1,61 +1,133 @@
 import './App.scss';
+import React, { useState } from 'react';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+import { User } from './types/User';
+import { TodoList } from './components/TodoList';
+
+function getUserById(userId: number): User | null {
+  const correctUser = usersFromServer.find(user => user.id === userId);
+
+  return correctUser || null;
+}
+
+function getUserByName(userName: string) {
+  const correctUser = usersFromServer.find(({ name }) => userName === name);
+
+  return correctUser || null;
+}
+
+export const preparedTodos: Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: getUserById(todo.userId),
+}));
 
 export const App = () => {
+  const [title, setTitle] = useState('');
+  const [todos, setTodos] = useState<Todo[]>(preparedTodos);
+  const [selectedUser, setSelectedUser] = useState('');
```

Comment:
will be better to handle users by id, not by name
because `id` it's main unique field for the users

Reply:
and in this case you don't need to use separate `getUserByName` function